### ISSUE TITLE
Disable hotkeys in fullscreen programs.

### ISSUE THREAD

**Issue by nightson** [0-don/clippy#5]
It would be helpful to have an option to disable all the hotkeys in fullscreen programs. You don't want to be brought to desktop when you're playing games :P

Thank you!

**Comment by 0-don** [comment]
there is only one hotkey running when the clipboard isnt beeing displayed

**Comment by nightson** [comment]
> there is only one hotkey running when the clipboard isnt beeing displayed

Sorry my bad. Did test enough. The default Ctrl+D is very easy to use but could easily be triggered when you're playing any game that utilizes a WASD control scheme.

**Comment by 0-don** [comment]
you can change that in the settings

**Comment by nightson** [comment]
> you can change that in the settings

I know that. But I want to use the default Ctrl+D. Could you add an option to disable the hotkey in fullscreen programs? Never mind if it's too much effort or a feature you don't like :P

**Comment by 0-don** [comment]
Yeah, it's not that easy since it's multiplatform, and I would have to implement each individually.

**Comment by nightson** [comment]
> Yeah, it's not that easy since it's multiplatform, and I would have to implement each individually.

Got it. It would also work if there is a hotkey or right-click menu option to disable the global hotkey. Okay, I admit that this isn't an elegant way to do it lol

**Comment by Mert-55** [comment]
Hi, I would like to give the issue a try.

**Comment by Mert-55** [comment]
Hi again, I would appreciate a review, thank you.
